Python/basic GUI calculator

```diff
@@ -0,0 +1,86 @@
+from tkinter import *
+
+def compute_handler():
+    a=int(temp.get())
+    b=int(tem.get())
+    f=a+b
+    result.set(f)
+
+
+def compute_handler2():
+    a=int(temp.get())
+    b=int(tem.get())
+    f=b-a
+    result.set(f)
+
+def compute_handler3():
+    a=int(temp.get())
+    b=int(tem.get())
+    f=a*b
+    result.set(f)
+
+def compute_handler4():
+    a=int(temp.get())
+    b=int(tem.get())
+    f=b/a
```

Comment:
What if I try to divide any number x by 0? You can use ```try-expect``` to handle such errors.

Reply:
will do .Thank you